vix/VixHost.py

```diff
@@ -1,6 +1,10 @@
 from __future__ import absolute_import
 
-from collections import namedtuple
+import collections
+import platform
+import os
+import os.path
+from xml.etree import ElementTree
```

Comment:
xml.etree.ElementTree is in [The Python Standard Library after Python 2.5](https://docs.python.org/2/library/xml.etree.elementtree.html).